gateway/main.go

```diff
@@ -18,19 +20,40 @@ func main() {
 
 	ctx := signals.Context()
 
-	{
-		address, token, opts, err := apiClientConfig()
-		if err != nil {
-			log.Fatal(err)
-		}
-		scrapeInterval, err := scrapeDuration()
-		if err != nil {
-			log.Fatal(err)
+	address, token, opts, err := apiClientConfig()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	apiClient := sdk.NewAPIClient(address, token, &opts)
+
+	noiseInterval, err := noiseFrequency()
```

Comment:
Nit: Reading through this, it was slightly jarring to switch from the "frequency" nomenclature to "interval." Let's stick with one.

```suggestion
	noiseFrequency, err := noiseFrequency()
```